addons/trenches/functions/fnc_handleDiggingServerSide.sqf

```diff
@@ -30,14 +30,25 @@ if (_initiator) then {
             params ["_args", "_handle"];
             _args params ["_trench", "_digTime"];
 
+            private _diggingPlayers = _trench getVariable [QGVAR(diggingPlayers), []];
+            {
+               if (isNull _x) then {
+                  _diggingPlayers deleteAt _forEachIndex;
```

Comment:
That won't work. You'll skil the element after the current.
How about `select` or simply `- [objNull]`